spec/std/option_parser_spec.cr

```diff
@@ -101,8 +101,8 @@ describe "OptionParser" do
     expect_capture_option ["--flag=123"], "--flag=FLAG", "123"
   end
 
-  it "has required option with = (3) raises" do
-    expect_missing_option ["--flag="], "--flag=FLAG", "--flag"
+  it "has required option with = (3) handles empty" do
+    expect_capture_option ["--flag="], "--flag=FLAG", ""
```

Comment:
I changed the behaviour of `OptionParser` here because I thought this behaviour was stupid.